packages/x-charts/src/BarChart/extremums.test.ts

```diff
@@ -55,8 +55,8 @@ describe('BarChart - extremums', () => {
 
       it('should correctly get Infinity when empty data', () => {
         const [x, y] = getExtremumX(buildData([], 'horizontal'));
-        expect(x).to.equal(null);
-        expect(y).to.equal(null);
+        expect(x).to.equal(Infinity);
+        expect(y).to.equal(-Infinity);
```

Comment:
Is there a special motivation for moving from `null` to `Infinity`?

Maybe adding [#13973](https://github.com/mui/mui-x/issues/13973) to the solved issues

Reply:
It simplifies the calculations and checks, however it doesn't seem to fix https://github.com/mui/mui-x/issues/13973

Reply:
Fix was simple though